app/views/guests/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1><%="Guest Name: #{@guest.name}"%></h1>
+
+<div id="rooms_stayed_in">
+  <%@guest.rooms.each do |room|%>
+    <p><%=("#{@guest.name} stayed in the #{room.suite} at #{room.hotel.name} for #{room.rate} per night.")%></p>
+  <%end%>
+</div>
+
+<div id="add_room_to_guest">
+  <%= form_with url: "/guest_rooms/#{@guest.id}", method: :post, local: true do |form| %>
```

Comment:
This path is ok, but I think it could be more RESTful if the `guest_id` were in a hidden field in the form. This would mean the end of the path matches with the `:post` method.